components/fonts/platform/windows/font.rs

```diff
@@ -107,39 +108,77 @@ impl PlatformFont {
             em_size,
             du_to_px: design_units_to_pixels,
             scaled_du_to_px: scaled_design_units_to_pixels,
+            variations,
         })
     }
+
+    fn new_with_variations(
+        font_face: FontFace,
+        pt_size: Option<Au>,
+        variations: &[FontVariation],
+    ) -> Result<Self, &'static str> {
+        if variations.is_empty() {
+            return Self::new(font_face, pt_size, vec![]);
+        }
+
+        // On FreeType and CoreText platforms, the platform layer is able to read the minimum, maxmimum,
+        // and default values of each axis. This doesn't seem possible here and it seems that Gecko
+        // also just sets the value of the axis based on the values from the style as well.
+        let variations: Vec<_> = variations
+            .into_iter()
+            .map(|variation| DWRITE_FONT_AXIS_VALUE {
+                axisTag: variation.tag.swap_bytes(),
```

Comment:
How was the need for `swap_bytes` determined?

Reply:
I've copy-and-pasted a comment about this here as well.